woocommerce/woocommerce
# [e2e tests] Add environment info rest route and environment reporter

### Changes proposed in this Pull Request:

Added a new rest route in the test_helper_apis plugins and used the info it provides in a new Playwright reporter that writes a file that can be interpreted by Allure.

### How to test the changes in this Pull Request:

Run the tests. Check the `test-results/allure-results/environment.properties` file. It should list the versions of core, php and the active plugins.
Check the Allure report. It should include the information from this file in the Overview page under the Environment section.
